manifest.json

```diff
@@ -14,6 +14,7 @@
     "open_in_tab": true
   },
   "permissions": [
+    "contextMenus",
```

Comment:
Luckily, this is all the permissions we need. We don't have to access all web pages.